vagrant-default.yml

```diff
@@ -442,10 +442,12 @@ php:
                 xdebug.remote_enable: On
                 xdebug.remote_connect_back: On
         php_fpm:
+            global:
+                pid: /run/php/php7.2-fpm.pid
             default:
                 user: vagrant
                 group: vagrant
-                listen: /var/run/php/default.sock
+                listen: /run/php/php7.2-fpm.sock
```

Comment:
please do not add version specific paths

Reply:
and it needs to be aligned with line 315

Reply:
probably we have to adapt the `PIDFile=/run/php/php7.2-fpm.pid` in the systemd config, i don't think the socket change is needed

Reply:
I see.

Yes, the socket does not need to be changed.

One solution would also be just to remove the `PIDFile` from `/etc/systemd/system/multi-user.target.wants/php7.X-fpm.service` completely. Should I go this way with this PR?

Reply:
depends what is easier to achieve:

add would be similar as here: https://github.com/vagrant-php/ubuntu/blob/18.04/ansible/roles/general/tasks/main.yml#L59-L64